fighthealthinsurance/views.py

```
@@ -192,23 +192,49 @@ def _persist_interested_professional_lead(
+        # instance is discarded afterwards — never save() it here, or an
+        # unauthenticated resubmission would overwrite the stored lead.
+        interested_pro.id = existing.id
+        notify_interested_professional(
+            interested_pro,
+            source=source,
+            subject=f"{subject_prefix} #{existing.id} (returning)",
+        )
+        if not existing.thankyou_email_sent:
+            try:
+                ThankyouEmailSender().dosend(interested_pro=existing)
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)